c2corg_ui/static/js/editing/routeediting.js

```diff
@@ -34,6 +34,13 @@ app.RouteEditingController = function($scope, $element, $attrs, $http,
     $uibModal, $compile, appLang, appAuthentication, ngeoLocation, appAlerts,
     appApi, authUrl, appDocument, appUrl, imageUrl) {
 
+
+  /**
+   * @type {angular.$compile}
+   * @private
+   */
+  this.compile__ = $compile;
```

Comment:
Is the double trailing "_" wanted?
In other files of this PR you use ``compile_`` with only one "_"?